pkgs/by-name/ni/niri/package.nix

```diff
@@ -110,6 +110,11 @@ rustPlatform.buildRustPackage (finalAttrs: {
         "-Wl,--pop-state"
       ]
     );
+
+    # Upstream recommends setting the commit hash manually when in a
+    # build environment where the Git repository is unavailable.
+    # See https://github.com/YaLTeR/niri/wiki/Packaging-niri#version-string
+    NIRI_BUILD_COMMIT = "Nixpkgs";
```

Comment:
nit: when is nixpkgs ever capitalized like this? i feel like it should be `"nixpkgs"` in lowercase, no?

Reply:
It's spelled "Nixpkgs" wherever people actually care about proper capitalization, such as in the [manual](https://nixos.org/manual/nixpkgs/unstable/#chap-stdenv)